media/css/firefox/family/components/_pro-tip.scss

```diff
@@ -28,6 +28,7 @@
 .c-pro-tip {
     background-color: f3.$black;
     font-weight: bold;
+    overflow-y: hidden;
```

Comment:
**issue (blocking)**: Safari still has horizontal scroll on the element
<img width="1402" alt="Screen Shot 2023-01-12 at 9 27 36 AM" src="https://user-images.githubusercontent.com/19650432/212030021-d6ec709b-c815-4bb2-af09-0df3048e94bb.png">

**suggestion**: based on local testing
```suggestion
    overflow-x: hidden;
```